docs/releases/handling-releases.md

```diff
@@ -158,7 +158,7 @@ _Outcome_: **Customers can install/update via WPORG; WPORG plugin page is up to
 
 -   Merge the release branch back into `main` (without the branch being up to date with `main`). This may have merge conflicts needing resolved if there are cherry-picked commits in the release branch.
 -   Restore the branch if it is deleted. Release branches are kept open for potential patch releases for that version.
--   For _major_ & _minor_ releases, update version on `main` with dev suffix, e.g. [`2.6-dev`](https://github.com/woocommerce/woocommerce-gutenberg-products-block/commit/e27f053e7be0bf7c1d376f5bdb9d9999190ce158).
+-   For _major_ & _minor_ releases, update version on `main` with dev suffix, e.g. [`2.6-dev`](https://github.com/woocommerce/woocommerce-gutenberg-products-block/commit/e27f053e7be0bf7c1d376f5bdb9d9999190ce158). **Note:** this should have already been taken care for you by the release deploy script, just verify that the versions have been updated correctly.
```

Comment:
Might be even clearer to remove `update` since this is automated. Something like: 

```
-   For _major_ & _minor_ releases, confirm version on `main` is correct and has dev suffix, e.g. [`2.6-dev`](https://github.com/woocommerce/woocommerce-gutenberg-products-block/commit/e27f053e7be0bf7c1d376f5bdb9d9999190ce158). 
**Note:** this is automated by the release deploy script, just verify that the versions have been updated correctly.
```

Reply:
As an aside, are you aware of the ["suggest" feature in GitHub](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request)? You can directly suggest the change in pull request review and then the author of the pull request just has to accept it to merge.